slang-kotlin/src/main/java/com/sonarsource/slang/kotlin/KotlinSensor.java

```diff
@@ -63,31 +67,28 @@ public void execute(SensorContext sensorContext) {
 
   private void analyseFiles(SensorContext sensorContext, Iterable<InputFile> inputFiles, List<TreeVisitor<InputFileContext>> visitors) {
     for (InputFile inputFile : inputFiles) {
+      InputFileContext inputFileContext = new InputFileContext(sensorContext, inputFile);
       try {
-        analyseFile(sensorContext, inputFile, visitors);
-      } catch (IllegalStateException e) {
-        // FIXME Assert errors from kotlin parser
-      } catch (Throwable e) {
-        // FIXME Assert errors from kotlin parser
+        analyseFile(inputFileContext, inputFile, visitors);
+      } catch (ParseException e) {
+        LOG.error("Parsing exception: " + e.getMessage());
```

Comment:
It would be better to give some information about the file and the location (if possible).